3] guessing game 2/main.py

```diff
@@ -0,0 +1,62 @@
+def get_answer() -> str:
+    """
+    A function for getting player's answer.
+
+    A function gets the player's answer and chcek if
+    it is in the list of correct answers. If it is, it 
+    return answer as string. If it is not, the function
+    starts again.
+    """
+    # getting an answer
+    answer = input("(ALLOWED ANSWERS: too low / too high / "
+                    "you quessed):\n")
+
+    # answer check
+    correct_answers = ["too low", "too high", "you quessed"]
+    if answer in correct_answers:
+        return answer
+    else:
+        print("Wrong value, try again...") 
+        get_answer()
```

Comment:
to delas rekurzivne jo? to je teda fikany :)

Reply:
Jj, učili mě, že se máme vyvarovat tomu, abychom psali stejný kód více krát, tak mi to přišlo nejjednodušší. Mám to přepsat?

Reply:
no to bys mela. anebo musis osetrit ze to nevyhodi eror. 

ale to nemusi opakovat. Staci to misto do rekurze dat do loopu preci